gltest/direct/sdk_loader.py

```
@@ -43,19 +53,47 @@ def parse_contract_header(contract_path: Path) -> Dict[str, str]:
-            version = final_url.split("/")[-1]
-            return version
+            releases = json.loads(resp.read().decode("utf-8"))
+        for release in releases:  # GitHub returns releases newest-first
+            if release.get("prerelease") or release.get("draft"):
+                continue
+            assets = release.get("assets", [])
+            if any(asset.get("name") == UNIVERSAL_ASSET for asset in assets):
+                return release["tag_name"]
     except Exception:
-        return "v0.2.12"
+        pass
```

Ruff (0.15.13)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)